apps/web/src/contexts/payment-context.tsx

```diff
@@ -718,7 +718,7 @@ export function usePaymentProvider({
 
   useEffect(() => {
     setPrice(release?.type === PackType.Auction ? bid : release?.price || 0)
-  }, [currency, bid]) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [bid]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I realize the eslint comment precedes these changes, but curious if it's still needed? Obviously the `currency` variable is not read in here, but `release` is and not listed. Not sure what the original reasoning was for disabling the rule.

Reply:
My assumption here is that we wouldn't want to re-run the `setPrice` function when release value is changed, only when the bid is

Reply:
went ahead and removed the disable in favor of explicitly listing the deps. My guess is they were omitted because they should never change.